docs/operational_guide/upgrading_m3.md

```diff
@@ -0,0 +1,101 @@
+# Upgrading M3
+
+## Overview
+
+This guide explains how to upgrade M3 from one version to another (e.g. from 0.14.0 to 0.15.0) - both on Kubernetes as well as on bare metal.
+This includes upgrading:
+
+- m3dbnode
+- m3coordinator
+- m3query
+- m3aggregator
+
+## m3dbnode
+
+### Graphs to monitor
+
+While upgrading M3DB nodes, it's important to monitor the status of bootstrapping the individual nodes. This can be monitored using the [M3DB Node Details](https://grafana.com/grafana/dashboards/8126) graph.
+Typically, the `Bootstrapped` graph under `Background Tasks` and the graphs within the `CPU and Memory Utilization` give a good understanding of how well bootstrapping is going.
+
+### Non-Kubernetes
+
+It is very important that for each replica set, only one node gets upgraded at a time. However, multiple nodes can be upgraded across replica sets. 
+
+1) Download new binary (linux example below).
+
+```bash
+wget "https://github.com/m3db/m3/releases/download/$VERSION/m3_$VERSION_linux_amd64.tar.gz" && tar xvzf m3_$VERSION_linux_amd64.tar.gz && rm m3_$VERSION_linux_amd64.tar.gz
+```
+
+2) Stop and upgrade one M3DB node at a time per replica set using the [systemd unit](https://github.com/m3db/m3/blob/master/integrations/systemd/m3dbnode.service).
+
+```bash
+# stop m3dbnode
+sudo systemctl stop m3dbnode
+
+# start m3dbnode with new binary
+./m3_$VERSION_linux_amd64/m3dbnode -f <config-name.yml>
```

Comment:
This step should say to replace your current `m3dbnode` binary with the one you just downloaded. With the current steps you're now running it outside of systemd, and run the risk of running 2 processes at once.

Reply:
Hmm where do you place the binary of m3db so that systemd can read from it?

Reply:
and then im guessing you do:
```
sudo systemctl start m3dbnode
```

Reply:
You can just say to replace it at the path specified in their [systemd unit](https://github.com/m3db/m3/blob/a1ccb6bc53e7b5962449434442f70125650400fe/integrations/systemd/m3dbnode.service). And re: command yup that's it